scripts/release_notes/commitlist.py

```diff
@@ -67,7 +69,9 @@ def read_from_disk(path) -> List[Commit]:
             reader = csv.DictReader(csvfile)
             rows = []
             for row in reader:
-                filtered_rows = {k: row[k] for k in commit_fields}
+                if row.get("new_title", "") != "":
+                    row["title"] = row["new_title"]
```

Comment:
This should be documented somewhere right? (I don't see it in the current doc yet, but we could expand that point on the template)